process_data_set_irene.R

```diff
@@ -0,0 +1,73 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+
+castle_solutions <- read_csv("data-raw/riddler-castles/castle-solutions.csv")
+
+castle_solutions<-castle_solutions%>% #clean the names
+  clean_names()
+colnames(castle_solutions)[11] <- "strategy"
+
+castle_solutions$user <- seq.int(nrow(castle_solutions)) #create user column
+  
+#gather the data to tidy form
+castle_solutions<-gather(castle_solutions, key = "castle", value = "number_of_soldiers", castle_1, castle_2, castle_3, castle_4, castle_5, castle_6, 
```

Comment:
Please add carriage returns so that this doesn't go so far on one line

Reply:
You could also the select helper `contains("castle_")` make this a little shorter.